sound_play/scripts/soundplay_node.py

```diff
@@ -188,6 +188,14 @@ def stopall(self):
         self.stopdict(self.filesounds)
         self.stopdict(self.voicesounds)
 
+    def update(self):
+        for sound in self.builtinsounds.values():
+            sound.update()
```

Comment:
cleanupdict() is already iterating through all of the sounds looking for things that need to be cleaned up; it seems like it would be better to call update there instead of introducing a new code path.